### PR TITLE
add omnisearch mock

### PR DESCRIPTION
**Pull request by robintiolie**

### CODE REVIEW

File: swagger.yaml [hunk 1]
```diff
@@ -1338,3 +1364,125 @@ definitions:
...
+      - title: "RM Makan Padang"
+        subtitle: "53 OUTLETS JAKARTA BARAT"
+        iconUrl: "https://www.gadjian.com/blog/wp-content/uploads/2018/01/Intip-Kesuksesan-Restoran-Padang-dengan-Sistem-Perjanjian-Kerja-Bagi-Hasil.jpg"
+        omniSearchType: "RESTAURANT_CHAIN"
+        chainData:
+          chainId: "2431"
+          geoDisplay:
+            name: "Menara Peninsula"
+            locationType: "GEO"
+            id: 7
```
Comment (mage95) [inline swagger.yaml]: better wrap it under generic `BasicSearchSpec` Field for easier navigation

File: swagger.yaml [hunk 2]
```diff
@@ -1338,3 +1364,125 @@ definitions:
...
+      - title: "Padang"
+        subtitle: "Cuisine"
+        omniSearchType: "TAGS"
+        tagData:
+          cuisineFilter:
+            - "PIZZA"
+            - "BURGER"
```
Comment (mage95) [inline swagger.yaml]: same

File: swagger.yaml [hunk 3]
```diff
@@ -1338,3 +1364,125 @@ definitions:
...
+      - title: "Kopi Kenangan"
+        subtitle: "Restaurant - Minuman"
+        iconUrl: "https://awsimages.detik.net.id/community/media/visual/2019/06/25/4b2e9e97-a47e-49d3-8e20-26074032b26f_11.jpeg?w=700&q=80"
+        omniSearchType: "RESTAURANT"
+        restaurantData:
+          restaurantId: "223"
```
Comment (mage95) [inline swagger.yaml]: idem

File: swagger.yaml [hunk 4]
```diff
@@ -1338,3 +1364,125 @@ definitions:
...
+      - title: "KFC Tanjung Duren"
+        subtitle: "Restaurant - Tanjung Duren"
+        iconUrl: "https://pbs.twimg.com/profile_images/3750210155/facf7e0f57bc8e7712f36b9a4e7d8bba_400x400.jpeg"
+        omniSearchType: "RESTAURANT"
+        restaurantData:
+          restaurantId: "726"
```
Comment (mage95) [inline swagger.yaml]: idem

**Comment by robintiolie** [comment]
should I wrapped the response inside object too?
data {
   results: List<Items>
}

now is 
data{
   List<Items>
}

**Comment by robintiolie** [comment]
we can discuss on monday

File: swagger.yaml [hunk 5]
```diff
@@ -1338,3 +1413,134 @@ definitions:
...
+    example:
+      data:
+        geoDisplay:
+          name: "Bandung"
+          id: 998
+          locationType: "GEO"
+        omniSearchTypes:
+        - "DISH"
+        - "CUISINE"
+        query: "blahhh blahh"
```
Comment (mage95) [inline swagger.yaml]: just sending query is fine since BE will return the matching data based on your query

Comment (robintiolie) [reply swagger.yaml]: types for filtering in glosarry page @mage95. for geoDisplay i think for MEP doesn't need

Comment (mage95) [reply swagger.yaml]: Glossary Search is out of scope of omni API, omni API purpose to search all data that we have in loka

Comment (robintiolie) [reply swagger.yaml]: i mean, omnisearch in glosarry page will return only DISH or CUISINE type. for more info https://29022131.atlassian.net/wiki/spaces/LOKA/pages/1107493212/Loka+Homepage+Glossary+Page+-+V0.
section 7.1 row 5

Comment (mage95) [reply swagger.yaml]: https://29022131.atlassian.net/wiki/spaces/LOKA/pages/1107137642/Loka+Search+Capabilities+-+V0 it seems there discrepancy on the spec

Comment (robintiolie) [reply swagger.yaml]: confirmed to Willy Aprianto actually

Comment (mage95) [reply swagger.yaml]: does this make it harder for the frontend to manage state of Autocomplete UI? if the approach like this? if not then LGTM but please add `GEO, RESTAURANT`

**Pull request merged**